src/components/ambassadorCard/ambassadorCard.module.scss

```
@@ -12,111 +12,99 @@
-    transition: $transition;
-    transition-property: filter;
-
-    &:hover {
-      filter: brightness(0.7);
+  &.birthday {
+    &::after {
+      content: url("../../assets/party.png");
+      width: 4rem;
+      height: auto;
+      position: absolute;
+      left: 50%;
+      top: 0;
+      transform: translate(-50%, -90%);
     }
   }
 
-  .img {
+  .hero {
+    position: relative;
     width: 100%;
-    aspect-ratio: 3;
-    object-fit: cover;
-    align-self: center;
-    border-radius: 0;
-    margin-bottom: 0.25rem;
 
-    @media (min-width: 480px) {
+    .img {
+      display: block;
+      width: 100%;
+      object-fit: cover;
       aspect-ratio: 2.2;
-    }
-  }
-
-  .name {
-    font-size: 1.3rem;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
-    max-width: calc(100% - 3rem);
-    width: 100%;
-    flex-shrink: 0;
-  }
 
-  .name,
-  .row {
-    margin: 0 0.5rem 0.25rem;
-  }
-
-  .info {
-    display: inline-flex;
-    align-items: center;
-    gap: 0.5rem;
-  }
-
-  .compact {
-    display: flex;
-    flex-wrap: wrap;
-    gap: 0.25rem 1.5rem;
-  }
-
-  .mod {
-    display: flex;
-    gap: 0.5rem;
-    align-items: center;
+      @media (min-width: 480px) {
+        aspect-ratio: 1.8;
+      }
 
-    img {
-      width: 1.5rem;
-      height: 1.5rem;
-      object-fit: cover;
-      border-radius: 0;
+      &:hover {
+        + .overlay {
+          opacity: 0.125;
+          backdrop-filter: blur(0.125rem);
+        }
+      }
     }
-  }
-
-  .site {
-    font-style: italic;
-    margin-top: 0.75rem;
 
-    a {
-      color: $secondary-color;
-      text-decoration: underline;
-      white-space: nowrap;
+    .overlay {
+      display: flex;
+      position: absolute;
+      top: 0;
+      width: 100%;
+      height: 2.25rem;
+      background: rgba($accent-color, 0.5);
+      backdrop-filter: blur(0.25rem);
       transition: $transition;
-      transition-property: color;
+      transition-property: opacity, backdrop-filter;
+
+      .name {
+        font-size: 1.3rem;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        overflow: hidden;
+        padding: 0 0.5rem;
+        align-self: center;
+        max-width: calc(100% - 2.5rem);
+        width: 100%;
+        flex-shrink: 0;
+      }
 
-      &:hover,
-      &:focus {
-        color: $tertiary-color;
+      .close {
+        position: absolute;
+        top: 50%;
+        right: 0.25rem;
+        transform: translateY(-50%);
+        width: 2rem;
+        display: block;
+        font-size: 1.5rem;
+        line-height: 1.1;
+        cursor: pointer;
+        text-align: center;
+        transition: $transition color;
+
+        &:hover {
+          color: $outline-color;
+        }
       }
     }
   }
 
   .scrollable {
+    display: flex;
+    flex-direction: column;
+    gap: 0.25rem;
     flex: 1 1 auto;
     overflow-y: auto;
-    margin-right: 1px;
+    margin: 0.25rem 0;
+    padding: 0 0.5rem 0.5rem;
```

```suggestion
    margin: 0;
    padding: 0.5rem 0.5rem 1rem;
```

I feel like the scrollable needs a bit more vertical padding at the top. also not sure why we had separate margin here?

---

Using margin for the vertical spacing so that it remains present as you scroll -- margin happens outside scroll, padding happens inside.